src/App.js

```diff
@@ -1,25 +1,11 @@
-import React from 'react';
-import logo from './logo.svg';
+//import React from 'react';
 import './App.css';
+//import Intro from './Components/Intro/Intro.js';
+import Def_linear from './Components/Def_linear/Def_linear.js';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    Def_linear()
```

Comment:
In React, instead of calling your component like a function, we insert it as an idiomatic declarative tag:
```suggestion
    <Def_linear />
```